packages/synapse-react-client/src/components/CreateTableViewWizard/TableTypeSelection.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import WizardChoiceButtonGroup from '../WizardChoiceButton/WizardChoiceButtonGroup'
+import WizardChoiceButton from '../WizardChoiceButton/WizardChoiceButton'
+import WizardChoiceButtonDescription from '../WizardChoiceButton/WizardChoiceButtonDescription'
+import { Link } from '@mui/material'
+
+const TABLE_DESCRIPTION = (
+  <>
+    <WizardChoiceButtonDescription my={1.25}>
+      Synapse tables are used to organize web-accessible, sharable, and
+      queryable data. Tables may be queried and edited with the Synapse UI, as
+      well as with the Synapse programmatic clients.
+    </WizardChoiceButtonDescription>
+    <Link
+      href={
+        'https://help.synapse.org/docs/Organizing-Data-With-Tables.2011038095.html'
+      }
+      target="_blank"
+      onClick={e => {
+        e.stopPropagation()
+      }}
+    >
+      More about Tables
+    </Link>
+  </>
+)
+
+const VIEW_DESCRIPTION = (
+  <>
+    <WizardChoiceButtonDescription my={1.25}>
+      Views allow you to see and query groups of other objects in Synapse, such
+      as files, tables, projects, or submissions and any associated annotations
+      about those items.
+    </WizardChoiceButtonDescription>
+    <Link
+      href={'https://help.synapse.org/docs/Views.2011070739.html'}
+      target="_blank"
+      onClick={e => {
+        e.stopPropagation()
+      }}
+    >
+      More about Views
+    </Link>
+  </>
+)
+
+export type TableTypeSelectionProps = {
+  /* Callback including the chosen type, and optional viewTypeMask if included in the choice (e.g. Project Views) */
+  onTableSelected: () => void
```

Comment:
Could this comment be misplaced? It seems to match `onTypeSelected` more than this function

Reply:
Yes, thanks for catching this. I originally had a different props signature and I changed it but forgot to change the comment.